src/Commands/Container/ContainerCheck.php

```diff
@@ -67,8 +66,6 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             ) {
                 try {
                     $this->getContainer()->get($serviceId);
-                } catch (NotImplementedException $notImplementedException) {
```

Comment:
What did you do here ?

Reply:
PHPStan reported this exception as never thrown. Is he wrong?